tests/components/zwave_js/test_device_condition.py

```diff
@@ -694,3 +697,39 @@ async def test_get_value_from_config_failure(
                 "endpoint": 10,
             },
         )
+
+
+async def test_entity_available_when_node_dead(
```

Comment:
Please move the new tests to an appropriate module. This module tests the device conditions feature. I'd put the tests in `test_init.py`.